types/aws-lambda/index.d.ts

```diff
@@ -41,6 +41,7 @@
 //                 Benoit Boure <https://github.com/bboure>
 //                 James Lakin <https://github.com/jamesorlakin>
 //                 Ross Gerbasi <https://github.com/aphex>
+//                 Otto Kruse <https://github.com/ottokruse>
```

Comment:
Be aware this will get you pinged for all PRs to this package, this might not be what you want

Reply:
Haha yeah maybe you are right ...
Removing myself now